src/analysis/normed_space/spectrum.lean

```diff
@@ -0,0 +1,117 @@
+/-
+Copyright (c) 2021 Jireh Loreaux. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jireh Loreaux
+-/
+import algebra.algebra.spectrum
+import analysis.calculus.deriv
+/-!
+# The spectrum of elements in a complete normed algebra
+
+This file contains the basic theory for the resolvent and spectrum of a Banach algebra.
+
+## Main definitions
+
+* `spectral_radius : ℝ≥0∞`: supremum of `∥k∥₊` for all `k ∈ spectrum 𝕜 a`
+
+## Main statements
+
+* `is_open_resolvent_set`: the resolvent set is open.
```

Comment:
I believe that for all these definitions and statements in the module docstring, you should use namespacing, `spectrum.is_open_resolvent_set`, etc.  This ensures that when the documentation is auto-generated, it will include a link to the declaration.